src/HomePage/components/Popup/Popup.tsx

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+
+const Popup = (props: any) => {
+    return (
+        <div className="popup-box">
+            <div className="box">
```

Comment:
Whenever we can reduce the amount of nested markup - we should do this. I am not sure you need div in div here.